harvard-lts/Harvard-Geospatial-Library
# Ruby upgrade

**Upgrades Ruby and makes docker compose consistent across main file and override**
* * *


# What does this Pull Request do?
Upgrades Ruby and makes docker compose consistent across main file and override (they weren't and it was making things not start right for ED who is a new builder).

# How should this be tested?

This is on dev if you want to look there for the ruby check.

To test the local version, pull this branch, and make sure to re-copy the override file since yours might vary locally. Follow the build steps (so do docker compose build and then the up line) and make sure things come up.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
